hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/TestHoodieTableSource.java

```diff
@@ -151,6 +159,91 @@ void testGetTableAvroSchema() {
     assertThat(schemaFields, is(expected));
   }
 
+  @Test
+  void readArrayOfRows() throws Exception {
```

Comment:
Can we also add a IT in `ITTestHoodieDataSource` ?

Reply:
Done.